src/templates/menu.mustache

```diff
@@ -1 +1 @@
-<li data-value="{{ key }}" class="lf-{{ key }} {{ cls }}">{{ str }}</li>
+<li data-value="{{ key }}" class="lf-{{ key }} {{ cls }}" tabindex="0" aria-label="{{ ariaLabel }} share">{{ str }}</li>
```

Comment:
Will https://github.com/Livefyre/streamhub-share/pull/18/files#diff-176014320dcab5ad64497002ad0433c3R131 put "Twitter share share" here?
If yes, then remove "share" from here; it'll be easier to maintain and easier for i18n if we keep use a whole statement instead of piecing it together.